src/frontend/src/hooks/finance.hooks.ts

```diff
@@ -15,3 +16,15 @@ export const useUploadSingleReceipt = (id: string) => {
     return data;
   });
 };
+
+/**
+ * custom react hook to get a single reimbursement request
+ * @param id Id of the reimbursement request to get
+ * @returns the reimbursement request
+ */
+export const useSingleReimbursementRequest = (id: string) => {
+  return useQuery<ReimbursementRequest, Error>(['reimbursement-requests', id], async () => {
```

Comment:
yeah we should make all these keys the same thing, so that these queries invalidate correctly, I'll update mine to be reimbursement-requests. It can have spaces tho for future reference